From 7d57eb79377e5d5889e0678cbda5ef1f7a4f7db4 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 08 Dec 2021 13:00:49 +0200
Subject: [PATCH] [SW-0] ppboot: rename elbi config func

As ELBI permission level set to SECURED in ppboot, so changed
the function name ppboot_xx() from preboot_xx().

Change-Id: I6bd7c651dec6c9c2e5d3595c1a2a064df04bf8b2
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index d6f66d4..f41418c 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -430,7 +430,7 @@
 		 mmPSOC_ARC1_MSTR_IF_CORE_LBW_AXPROT_OVRD);
 }
 
-static void preboot_secure_elbi(void)
+static void ppboot_secure_elbi(void)
 {
 	/* make transactions via ELBI as secured */
 	regmap_writel(AXPROT_OVRD_SEC, mmPCIE_ELBI_RR_MSTR_IF_CORE_LBW_AXPROT_OVRD);
@@ -865,7 +865,7 @@
 
 void ppboot_security_init(void)
 {
-	preboot_secure_elbi();
+	ppboot_secure_elbi();
 
 	/* Initialize only if security is enabled */
 	if (!preboot_security_is_enabled())
